src/main/kotlin/no/nav/familie/tilbake/oppgave/OppgavePrioritetService.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.familie.tilbake.oppgave
+
+import no.nav.familie.kontrakter.felles.oppgave.Oppgave
+import no.nav.familie.kontrakter.felles.oppgave.OppgavePrioritet
+import no.nav.familie.tilbake.config.FeatureToggleConfig
+import no.nav.familie.tilbake.config.FeatureToggleService
+import no.nav.familie.tilbake.kravgrunnlag.KravgrunnlagRepository
+import no.nav.familie.tilbake.kravgrunnlag.domain.Klassetype
+import no.nav.familie.tilbake.kravgrunnlag.domain.Kravgrunnlag431
+import org.springframework.stereotype.Service
+import java.math.BigDecimal
+import java.util.UUID
+
+@Service
+class OppgavePrioritetService(
+    private val kravgrunnlagRepository: KravgrunnlagRepository,
+    private val featureToggleService: FeatureToggleService
+) {
+
+    fun utledOppgaveprioritet(behandlingId: UUID, oppgave: Oppgave? = null): OppgavePrioritet {
+        val finnesKravgrunnlag = kravgrunnlagRepository.existsByBehandlingIdAndAktivTrue(behandlingId)
+
+        return if (finnesKravgrunnlag && featureToggleService.isEnabled(FeatureToggleConfig.SETT_PRIORITET_PÅ_OPPGAVER)) {
+            val kravgrunnlag = kravgrunnlagRepository.findByBehandlingIdAndAktivIsTrue(behandlingId)
+
+            val feilutbetaltBeløp = utledFeilutbetaling(kravgrunnlag)
+
+            when {
```

Comment:
Gjelder disse grensene for alle type stønader (BA)?

Reply:
Vi sjekker med BA før prodsetting :)

Reply:
"Vegard" mente det, ifølge Hilde. Men nå vet jeg ikke hvem Vegard er - og om han har peiling på BA / KS :) Så vi får spørre de.

Reply:
BA tar gjerne denne endringen, men kan nok hende de vil "skru" på tallene for sin del.